### PR TITLE
Renamed German category to 'Müsli'

### PR DESCRIPTION
**Pull request by KommX**
Renamed German category to 'Müsli', this is the word everyone in Germany uses, see https://de.wikipedia.org/wiki/M%C3%BCsli

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -40566,7 +40566,7 @@ bg:Мюсли
 ca:Musli
 cs:müsli
 da:Müsli
-de:Müesli, Müslis
+de:Müsli, Müesli
```
Comment (hangy) [inline taxonomies/categories.txt:40569]: We kinda want plurals as the primary name, where they exist. Maybe use Müsli as an alias?

Comment (KommX) [reply taxonomies/categories.txt:40569]: I don't know too much about the inside of openfoodfacts, but I hoped to achieve that I can select "Müsli" (or "Müslis", I don't care) as a category. Right now, I can't do that, and I have to enter "Müesli" which is a word I've never heard before.

Comment (hangy) [reply taxonomies/categories.txt:40569]: Yep, me neither. Never heard of that spelling. 🤷‍♂️

Comment (svensven) [reply taxonomies/categories.txt:40569]: blame the swiss, that wiki link above says "...in der Schweiz und im südlichen Baden-Württemberg Müesli..." :)

Comment (KommX) [reply taxonomies/categories.txt:40569]: So, I changed the names to their plural forms in an additional commit. I hope, I did it right.

Comment (svensven) [reply taxonomies/categories.txt:40569]: looks good. thanks :)

**Review by svensven — APPROVED** [review]

**Pull request merged**